docs/MIGRATING_TO_1_0.md

```diff
@@ -0,0 +1,29 @@
+# Migrating from 0.3 to 1.0
+
+## Chained operations
+[`chain()`](https://zenika.github.io/immutadot/immutadot/0.3/seq.html#.chain) has been removed in favor of [`flow()`](https://zenika.github.io/immutadot/immutadot/1.0/core.html#.flow).
+
+`chain()` had some major drawbacks :
+ - Imports the whole of immutadot
+ - Is difficult to extend
+
+Migrating from `chain()` to `flow()` is pretty easy :
+
+```diff
```

Comment:
Waouh nice ! Didn't know that diff rendering was available in flavored markdown ❤️